freqtrade/freqai/data_kitchen.py

```diff
@@ -1060,3 +1068,35 @@ def get_unique_classes_from_labels(self, dataframe: DataFrame) -> None:
         if self.unique_classes:
             for label in self.unique_classes:
                 self.unique_class_list += list(self.unique_classes[label])
+
+    def save_backtesting_prediction(
+        self, file_name: str, root_folder: str, append_df: DataFrame
+    ) -> None:
+
+        """
+        Save prediction dataframe from backtesting to h5 file format
+        :param file_name: h5 file name
+        :param root_folder: folder to save h5 file
+        """
+        backtesting_root = Path(
+            self.full_path
+            / root_folder
+        )
+        if not backtesting_root.is_dir():
+            backtesting_root.mkdir(parents=True, exist_ok=True)
+
+        full_file_path = Path(self.full_path / root_folder / file_name)
+        append_df.to_hdf(full_file_path, key='append_df', mode='w')
+
+    def get_backtesting_prediction(
+        self, root_prediction: str, prediction_file_name: str
+    ) -> DataFrame:
+        """
+        Retrive from disk the prediction dataframe
+        :param prediction_file_name: prediction file full path
+        :return:
+        :Dataframe: Backtesting prediction from current backtesting period
+        """
+        prediction_path = Path(self.full_path / root_prediction / prediction_file_name)
```

Comment:
Similar here, we can just use the `dk.backtesting_results_path` and `dk.model_filename` directly instead of passing arguments around.

Reply:
Thanks for the feedback. Fixed!